wremnants/theory_tools.py

```diff
@@ -242,7 +242,7 @@ def build_weight_expr(df, exclude_weights=[]):
     valid_cols = df.GetColumnNames()
     if nominal_weight not in valid_cols:
         raise ValueError(f"The weight '{nominal_weight}' must be defined in the histmaker!")
-    weights = ["central_pdf_weight", "theory_corr_weight", "exp_weight"]
+    weights = [nominal_weight, "central_pdf_weight", "theory_corr_weight", "exp_weight"]
```

Comment:
Could we rename this to "base_weight" to avoid confusion with the "nominal_weight" column?

Reply:
Or just use the string "weight" two times